parser/parser.y

```diff
@@ -12618,19 +12624,20 @@ CommaOpt:
  *  https://dev.mysql.com/doc/refman/5.7/en/account-management-sql.html
  ************************************************************************************/
 CreateUserStmt:
-	"CREATE" "USER" IfNotExists UserSpecList RequireClauseOpt ConnectionOptions PasswordOrLockOptions CommentOrAttributeOption
+	"CREATE" "USER" IfNotExists UserSpecList RequireClauseOpt TokenRequireOpt ConnectionOptions PasswordOrLockOptions CommentOrAttributeOption
```

Comment:
I'm not very sure: why not specify the token options in the `RequireClauseOpt` too?

Reply:
The `RequireClauseOpt` is defined as an array of `TLSOption`, but the token options are not declared for TLS.
```text
RequireClauseOpt:
	{
		$$ = []*ast.TLSOption{}
	}
|	RequireClause
```